app/models/medicaid/export_factory.rb

```diff
@@ -0,0 +1,28 @@
+module Medicaid
+  class ExportFactory
+    DELAY_THRESHOLD = 30
+
+    def self.create(params)
+      new.create(params)
+    end
+
+    def create(params)
+      enqueue(Export.create(params))
+    end
+
+    def enqueue(export)
+      export.transaction do
+        export.save!
+        case export.destination
+        when :office_email
+          Medicaid::OfficeEmailApplicationJob.perform_later(export: export)
+        else
+          raise UnknownExportTypeError, export.destination
```

Comment:
When would this happen?

Reply:
when someone writes invalid code but the error gets lost because we just don't catch it in a case statement

Reply:
seems like a testing concern rather than a app code concern but nbd. lgtm